### PR TITLE
add blog post to point to "Decentralization Unchained" event

### PR DESCRIPTION
**Pull request by r10s**

### CODE REVIEW

File: _posts/2018-10-28-decentralization-unchained.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+After the "Delta-X" Event in Freiburg this summer, 
+there is now "Decentralization Unchained" -
+**one day of talks, workshops, discussions, hacking**
+around the topics of decentralizing infrastructures in common, 
+and, of course, Delta.Chat in special.
+
+"Decentralization Unchained" happens in Kyiv, November 3, 2018.
+
+For the whole prorgram and further information,
```
Comment (ralphtheninja) [inline _posts/2018-10-28-decentralization-unchained.md]: program

**Review by ralphtheninja — CHANGES_REQUESTED** [review]

File: _posts/2018-10-28-decentralization-unchained.md (hunk 1 above)
Comment (r10s) [reply _posts/2018-10-28-decentralization-unchained.md]: thanks :)

File: _posts/2018-10-28-decentralization-unchained.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Decentralization Unchained"
+author: bjoern
+---
+
+After the "Delta-X" Event in Freiburg this summer, 
+there is now "Decentralization Unchained" -
+**one day of talks, workshops, discussions, hacking**
+around the topics of decentralizing infrastructures in common, 
+and, of course, Delta.Chat in special.
```
Comment (okdistribute) [inline _posts/2018-10-28-decentralization-unchained.md]: in special -> especially

File: _posts/2018-10-28-decentralization-unchained.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+"Decentralization Unchained" happens in Kyiv, November 3, 2018.
+
+For the whole program and further information,
+also about how to contribute yourself,
```
Comment (okdistribute) [inline _posts/2018-10-28-decentralization-unchained.md]: "about how you can contribute," 
the other way sounds like some sort of human sacrifice/cannibalism

Comment (ralphtheninja) [reply _posts/2018-10-28-decentralization-unchained.md]: > the other way sounds like some sort of human sacrifice/cannibalism

:laughing:

**Review by ralphtheninja — APPROVED** [review]

**Pull request merged**